common/student/activation.ts

```diff
@@ -57,6 +58,7 @@ export async function deactivateStudent(
         },
         include: {
             course_instructors_student: true,
+            subcourse: true,
```

Comment:
Shouldn't we query subcourses here instead? We actually care about "subcourse instructors" not "course instructors" - this code still assumes there is a one to one mapping between them ...